test-projects/aio-v8-single-worker-bad-config/src/dx-asset-compute-worker-1/ext.config.yaml

```diff
@@ -0,0 +1,21 @@
+operations:
```

Comment:
qs, why is this config bad?

Reply:
Line 15 points to a file that doesn't exist